### PR TITLE
mark classes implementing the `WarmableInterface` as `final`

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 7.1
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | 
| License       | MIT

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/CacheWarmer/AbstractPhpFileCacheWarmer.php [hunk 1]
```diff
@@ -17,6 +17,9 @@
 use Symfony\Component\Config\Resource\ClassExistenceResource;
 use Symfony\Component\HttpKernel\CacheWarmer\CacheWarmerInterface;
 
+/**
+ * @internal since Symfony 7.1
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/CacheWarmer/AbstractPhpFileCacheWarmer.php]: I'm not sure about this one, the class can be useful for third party bundles. We already made a method final/internal so we already closed the class a bit.

Comment (xabbuh) [reply src/Symfony/Bundle/FrameworkBundle/CacheWarmer/AbstractPhpFileCacheWarmer.php]: Should we mark the `warmUp()` method as final instead?

Comment (nicolas-grekas) [reply src/Symfony/Bundle/FrameworkBundle/CacheWarmer/AbstractPhpFileCacheWarmer.php]: I'm not sure it would provide much benefits doing so.

Comment (nicolas-grekas) [reply src/Symfony/Bundle/FrameworkBundle/CacheWarmer/AbstractPhpFileCacheWarmer.php]: up to revert this line @xabbuh?

Comment (xabbuh) [reply src/Symfony/Bundle/FrameworkBundle/CacheWarmer/AbstractPhpFileCacheWarmer.php]: done

**Comment by nicolas-grekas** [comment]
Thank you @xabbuh.

**Pull request merged**